tasks/coverage/typescript.snap

```diff
@@ -97,15 +97,13 @@ Expect to Parse: "async/es5/asyncArrowFunction/asyncArrowFunction8_es5.ts"
    ╰────
 Expect to Parse: "async/es5/asyncClass_es5.ts"
 
-  × Automatic Semicolon Insertion
+  × Unexpected token
    ╭─[async/es5/asyncClass_es5.ts:3:1]
  3 │ // @noEmitHelpers: true
  4 │ async class C {
-   ·      ┬
-   ·      ╰── Expected a semicolon or an implicit semicolon after a statement, but found none
+   · ─────
  5 │ }
```

Comment:
This PR turns a lot of `Automatic Semicolon Insertion` errors into `Unexpected token`, which is good. Because now `async class C {` is recognized as a declaration rather than an expression followed by a class declaration.

Reply:
Nice!